qemu/tests/cfg/block_with_iommu.cfg

```diff
@@ -11,15 +11,15 @@
     variants:
         - verify_enabled:
             only Linux
-            only RHEL.8
-            only Host_RHEL.m8
+            no RHEL.7
+            no Host_RHEL.m7
             only virtio_scsi
             virtio_dev_disable_legacy = on
             virtio_dev_disable_modern = off
             check_key_words = "DMAR: IOMMU enabled;"
             check_key_words += "DMAR: Intel(R) Virtualization Technology for Directed I/O"
             variants:
-                - @default:
+                - default:
```

Comment:
I think show variants in test is better, it does not relate to this title.

Reply:
Would you like to give more detail about the reason `better`?
From my side, I think the variant `default` we don't need to show explicitly since it doesn't have a specific meaning, it may be just like a placeholder here. 
And we should give an explanation about this variant `default` if we want to show it explicitly.
Thanks.

Reply:
I care about the real full case name more than its' meaning. 
I will revert the change since it is irrelevant to this patch.